src/packages/core/src/server.ts

```diff
@@ -43,13 +51,13 @@ export type Callback = (err: Error | null) => void;
  * Server ready state constants.
  *
  * These are bit flags. This means that you can check if the status is:
- *  * ready: `status === Status.ready` or `status & Status.ready !== 0`
- *  * opening: `status === Status.opening` or `status & Status.opening !== 0`
- *  * open: `status === Status.open` or `status & Status.open !== 0`
- *  * opening || open: `status & Status.openingOrOpen !== 0` or `status & (Status.opening | Status.open) !== 0`
- *  * closing: `status === Status.closing` or `status & Status.closing !== 0`
- *  * closed: `status === Status.closed` or `status & Status.closed !== 0`
- *  * closing || closed: `status & Status.closingOrClosed !== 0` or `status & (Status.closing | Status.closed) !== 0`
+ *  * ready: `status === ServerStatus.ready` or `status & ServerStatus.ready !== 0`
+ *  * opening: `status === ServerStatus.opening` or `status & ServerStatus.opening !== 0`
+ *  * open: `status === ServerStatus.open` or `status & ServerStatus.open !== 0`
+ *  * opening || open: `status & ServerStatus.openingOrOpen !== 0` or `status & (ServerStatus.opening | ServerStatus.open) !== 0`
+ *  * closing: `status === ServerStatus.closing` or `status & ServerStatus.closing !== 0`
+ *  * closed: `status === ServerStatus.closed` or `status & ServerStatus.closed !== 0`
+ *  * closing || closed: `status & ServerStatus.closingOrClosed !== 0` or `status & (ServerStatus.closing | ServerStatus.closed) !== 0`
```

Comment:
This is just a drive-by fix up. A version of this PR used `ServerStatus`, and I had tried to copy paste from the examples here... but it didn't work because the names were wrong.

Let me know if you want this in another PR.